chimney/src/main/scala/io/scalaland/chimney/TransformationError.scala

```diff
@@ -0,0 +1,6 @@
+package io.scalaland.chimney
+
+case class TransformationError[M](message: M, errorPath: List[ErrorPathNode] = Nil) {
+  def prepend(node: ErrorPathNode): TransformationError[M] =
+    TransformationError[M](message, errorPath.prepended(node))
+}
```

Comment:
Can we add some method that returns nicely formatted string (like: `"foo.bar[5].baz"`)?

Reply:
Added